fipy/__init__.py

```diff
@@ -35,8 +35,37 @@
 from builtins import input
 __docformat__ = 'restructuredtext'
 
+import os
+
+# configure logging before doing anything else, otherwise we'll miss things
+if 'FIPY_LOG_CONFIG' in os.environ:
+    import json
+    import logging.config
```

Comment:
logging is part of the standard library so maybe could be put to the top of the file

Reply:
I s'pose